packages/instrumentation/src/__tests__/ActiveTracingHelper.test.ts

```diff
@@ -0,0 +1,148 @@
+import { trace } from '@opentelemetry/api'
```

Comment:
We'd prefer to test this over as a functional test, rather than unit test with a lot of mocking, so we can be sure this works with the rest of the OTEL stack.

You could add a test to an existing `tracing` test [here](https://github.com/prisma/prisma/blob/b333dcd4979d65dc34036d07f998566fb6f520c9/packages/client/tests/functional/tracing/tests.ts#L87) and documentation on our functional tests [here](https://github.com/prisma/prisma/blob/main/TESTING.md#prisma-client-functional-tests)